### PR TITLE
Bug in seeInRepository when using @Embedded

### PR DESCRIPTION
**Pull request by jachim**
Related to issue #2380

### CODE REVIEW

File: src/Codeception/Module/Doctrine2.php [hunk 1]
```diff
@@ -424,7 +424,7 @@ protected function buildAssociationQuery($qb, $assoc, $alias, $params)
             if ($val === null) {
                 $qb->andWhere("s.$key IS NULL");
             } else {
-                $paramname = "s__$key";
+                $paramname = str_replace(".", "", "s_$key");
```
Comment (DavertMik) [inline src/Codeception/Module/Doctrine2.php]: I didn't work with Doctrine for a while but why `s__` is changed to `s_`. Is that ok?

Comment (jachim) [inline src/Codeception/Module/Doctrine2.php]: When you are working with Embeddable object in Doctrine you must use notation with dot, for example:
$I->seeInRepository("Post", [
    'author.email' => "email@example.com",
]);

A variable $paramname is build based on $key so in above example will be: "s__author.email". A dot in this variable is a reason for exception:
[Doctrine\ORM\Query\QueryException] [Syntax Error] line 0, col 93: Error: Expected end of string, got '.' 

With this replace everything works great, I haven`t noticed any problems. I see all checks have passed so I hope you will merge this pull request.

**Comment by raistlin** [comment]
Seems safe enough to be merged

**Comment by DavertMik** [comment]
Right now we don't have Doctrine-related tests in core, we use our fork of [symfony-demo app](http://github.com/Codeception/Symfony-demo) for it. @jachim  If you will time, could you add a test there to prove embeddeds work fine? 

But anyway, I'm merging it.

**Pull request merged**